install_config/configuring_sdn.adoc

```diff
@@ -211,6 +211,50 @@ Check VNIDs by running:
 $ oc get netnamespace
 ----
 
+[[migrating-between-sdn-plugins-networkpolicy]]
+=== Migrating from the ovs-multitenant Plugin to ovs-networkpolicy
+
+If you want to migrate from *ovs-multitenant* to *ovs-networkpolicy*, you will first need
+to make sure that every namespace has a unique `NetID`; if you have previously
+xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined projects
+together] or xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made
+projects global], you will need to undo that before switching to the *ovs-networkpolicy*
+plugin, or NetworkPolicy objects may not function correctly.
+
+A helper script is available that will take care of this, and also create NetworkPolicy
+objects to isolate previously-isolated namespaces, and allow connections between
+previously-joined namespaces.
+
+. Download https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh
++
+----
+$ curl https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh
+$ chmod a+x migrate-network-policy.sh
+----
+. While still running the *ovs-multitenant* plugin (that is, before running any of the
+migration steps above), run that script as a user with cluster administrator rights on
+the OpenShift cluster
++
+----
+$ ./migrate-network-policy.sh
+----
+
+After running the script, every project will now be fully isolated from every other
+project, so connection attempts between pods in different projects will fail until you
+complete the migration to the *ovs-networkpolicy* plugin.
+
+The script will also set up
+xref:../admin_guide/managing_networking.adoc#admin-guide-networking-networkpolicy-setting-default[default
+NetworkPolicy objects] to be created in any newly-created projects so that they will have
```

Comment:
actually, this part isn't quite right; the user will have to take additional steps here. But you can review the rest of it while I figure that out